internal/jet/expression.go

```diff
@@ -316,6 +316,32 @@ func (s *complexExpression) serialize(statement StatementType, out *SQLBuilder,
 	}
 }
 
+type arraySubscriptExpression struct {
```

Comment:
This is the old way to construct custom expression. The new and easier way is using `CustomExpression` and `Token`. In this case: `CustomExpression(array, Token("["), index, Token("]"))`.

Reply:
Done